frontend/packages/dev-console/features/pipeline-Add_pages_for_tekton_trigger

```diff
@@ -0,0 +1,96 @@
+   When user clicks on kebab menu of the pipeline
+   And user selects Remove Trigger
+   And remove Trigger dialog box opens.
+   When user selects Trigger to be removed in the dropdown
+   And user clicks Remove
+   And user goes to pipeline details page.
+   Then user can not see that Trigger template and event listener.
+
+
+@regression
+Scenario: View Pipeline resources throughout the related types
+   Given user is in the Administrator perspective
+   When user clicks pipeline under Pipelines tab
+   And user navigates to Pipelines via the tabs on the top of the page
+   And the Pipelines page contains the Pipelines Name, Last Run, Last Run Status, and Last Started columns
+   And user navigates to Pipeline Runs via the tabs on the top of the page
+   And the Pipelines Runs page contains the Name, Status, Started, and Duration columns.
+   And user navigates to Pipeline Resources 
+   And the Pipelines Resources page contains the Name, Created, and Type columns.
+   And user navigates to Conditions via the tabs on the top of the page
+   Then the Conditions page contains the Name and Created columns.
+
+@regression
+Scenario: Create option in pipeline page
+   Given user is in the Administrator perspective
+   When user clicks pipelines under Pipelines tab
+   And the user click on the Create button from top right corner.
+   And the action item is to create Pipelines.
+   And the action item is to create Pipeline Resources.
+   Then the action item is to create Conditions
+
+@regression
+Scenario: View Pipeline task resources
+   Given user is in the Administrator perspective
+   When user clicks Tasks under Pipelines tab
+   And user navigates to Tasks
+   And the Tasks page contains the Name and Created columns.
+   And user navigates to Task Runs
+   And the Task Runs page contains the Name, Status, Started, and Duration columns.
+   And user navigates to Cluster Tasks
+   Then the Cluster Tasks page contains the Name and Created columns.
+
+@regression
+Scenario: Create option in task page
+   Given user is in Administrator perspective
+   When user clicks task under Pipelines tab
+   And the user click on the Create button.
+   And the action item is to create Task.
+   And the action item is to create Task Run.
+   Then the action item is to create Cluster Tasks.
+
+@regression
+Scenario: View Pipeline triggers resources
+   Given User is in Administrator perspective
+   When User clicks Triggers under Pipelines tab
+   And User navigates to Event Listeners
+   And the Event Listeners page contains the Name and Created columns.
+   And user navigates to Trigger Templates
+   And the Trigger Templates page contains the Name, Created, and Type columns.
+   And user navigates to Trigger Bindings
+   And the Trigger Bindings page contains the Name and Created columns.
+   And user navigates to Cluster Trigger Bindings
+   Then the Cluster Trigger Bindings page contains the Name and Created columns.
```

Comment:
Can you split this scenario in multiple like one for event listener, one for trigger templates, one for trigger bindings and one for cluster trigger bindings with same given statement? It will be easy for the validation purpose. You have three when statements without any actions performing and that can be converted into then statement.
e.g. 1. And the Event Listeners page contains the Name and Created columns. (It can be then statement if you create different scenario for even listeners.)
Same with other three as well.

Reply:
I didn't create separate because it was an unnecessary effort as these can be validated as part of a single scenario